src/plugins/intel_cpu/thirdparty/llmdnn/src/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 3.18)
```

Comment:
- why copyright header is missed here?
- please, decrease minimal cmake version. OpenVINO have 3.13 as minimal supported.

Reply:
Done.